protos/sources/vega/commands/v1/commands.proto

```diff
@@ -90,8 +90,8 @@ message OrderAmendment {
   optional int64 expires_at = 5;
   // Amend the time in force for the order, set to TIME_IN_FORCE_UNSPECIFIED to remain unchanged.
   vega.Order.TimeInForce time_in_force = 6;
-  // Amend the pegged order offset for the order
-  // This field is an unsigned integer passed as a string and needs to be scaled using the market's decimal places.
+  // Amend the pegged order offset for the order. This field is an unsigned integer passed as a string
+  // and needs to be scaled using the market's decimal places.
```

Comment:
Instead of saying `and needs to be scaled` can we not just say `in the market's precision`. So something like:
```
  // Amend the pegged order offset for the order. Must represent an unsigned integer in the market's precision.
```